admin/pages/settings.php

```diff
@@ -80,6 +81,9 @@
                 <label>PWA App Endpoint</label>
                 <input type="text" name="appEndpoint" value="<?= $theme->getAppEndpoint() ?>" />
                 <div class="spacer-20"></div>
+
+                <input type="checkbox" style="margin: 0 10px 0 0px;" name="showClassicSwitch" <?= $theme->getShowClassicSwitch() ? 'checked' : '' ?> /> Show Classic Site Switch
```

Comment:
2012 Called, they want their in-line styles back.
```suggestion
                <input type="checkbox" name="showClassicSwitch" <?= $theme->getShowClassicSwitch() ? 'checked' : '' ?> /> Show Classic Site Switch
```